src/Adapter/Requirement/CheckMissingOrUpdatedFiles.php

```diff
@@ -45,7 +45,7 @@ public function getListOfUpdatedFiles($dir = null, $path = '')
         );
         
         if (is_null($dir)) {
-            $xml = @simplexml_load_file(_PS_API_URL_.'/xml/md5/'.AppKernel::VERSION.'.xml');
+            $xml = @simplexml_load_file(_PS_API_URL_.'/xml/md5-'.AppKernel::MAJOR_VERSION.'/'.AppKernel::VERSION.'.xml');
```

Comment:
Hum... Can someone explain me why they change `xml/md5` in `xml/md5-17`? oO when we use the version in the name of the file. :cold_sweat:

Reply:
ping @eternoendless :)

Reply:
It has been like that since the start of 1.7 as far as I know

Reply:
BTW this will come back to bite us in the back if we ever get to PrestaShop v17.0 :P

Reply:
:cry:  Is their a way to keep `xml/md5-17` and move to `xml/md5`, sure we will forget to move to `xml/md5-18` :)

Reply:
@eternoendless I'd like to read PrestaShop 7 instead, isn't prettier? :)